index.js

```diff
@@ -100,11 +100,20 @@ const displayPost = (post) => {
     <td class="post-posters">
       <div class="postersAvatars">${postersAvatars}</div>
     </td> 
-    <td class="post-replies">${post.posts_count-1}</td>
+    <td class="post-replies">${post.posts_count - 1}</td>
     <td class="post-views"></td>
     <td class="post-activity">${formatDateDiff(Date.now(), post.bumped_at)}</td>
   </tr>`;
   postsContainer.innerHTML += postRow;
 };
 
 copyright.innerText = new Date().getFullYear();
+
+let sortBtns = document.getElementsByName("sort");
+for (let btn of sortBtns) {
+  btn.addEventListener("click", handleSortBtnClick);
+}
```

Comment:
```suggestion
sortBtns.forEach(btn => {
  btn.addEventListener("click", handleSortBtnClick);
}
```

Reply:
explicit in the sense?
You are right.  Using ES6 is the better practice, but I loose my sense of logic if I try to use ES6. I get lost in the syntax, I I guess! But maybe once I have written the code, I can refactor it.

Reply:
It is more clear what the code does with calling forEach. Even when you read it.
sortBtns for each btn.. That you are doing something with all of them.
As I said, not a big deal. You could see the difference better if you were using Array and accessing its elements by index.

For..of loop was actually implemented as ES6 feature if I remember correctly 😅

You don't need to use the arrow function. You could write function(btn) instead.

Reply:
Oh!  I thought it  for...of came with ES6. But the arrow style definition writing is ES6, right?

Reply:
- It is better to use a method on the object if you can and when you know that you want to iterate through all items. Like using `forEach` in this case.
- If you know ahead of time that you want to possibly iterate through all of them but may need to skip some items or stop iterating once some condition is met, you better use `for` loop.
- If you want something to be happening repeatedly until some conditions are not met, you better use `while` loop.

You cannot use `break` of `continue` statements inside `forEach` but you can in the other two.